tests/form.js

```diff
@@ -361,7 +361,7 @@ define([
                 [{id: "old", src: "new://3"}, "old-1", "src"],
                 [{id: "old", src: "old://"}, "old", "src"],
                 [{id: "any", src: "old://"}, "old", "src"],
-                [{id: "known", src: null}, "known", "known://"],
+                [{id: "known", src: undefined}, "known", "known://"],
```

Comment:
> The defaults function now only overrides undefined values, not null ones.

ha, this was the one change that caused problems when upgrading underscore in HQ's app manager